README.md

```diff
@@ -14,30 +14,49 @@ This repo represents the Community Automation effort where teams can contribute
 
 A public docker image has been create with all prereq's for running playbooks.
 
-**MAC/RHEL7** users need to install docker to run this option
+Make sure docker or podman(RHEL 8) is installed and running on your workstation
 
 **NOTES:**
 
-- On RHEL8 use podman
 - You may need to do a **docker logout** before you begin.
 - default ssh keys are comming from your ~/.ssh/id_rsa and ~/.ssh/id_rsa.pub
 
 **Tested on MAC (big sur), Ubunutu 16.04,18.04,20.04, and RHEL 7/8**
 
+## MAC, Ubuntu, and RHEL 7 users
+
```

Comment:
I think what your doing here is creating a local environment to run in, might want to state that this is what is being created with this option.

Reply:
done